lib/omnibus/git_cache.rb

```diff
@@ -32,7 +32,7 @@ class GitCache
     # will not have the generated content, so these snapshots would be
     # incompatible with the current omnibus codebase. Incrementing the serial
     # number ensures these old shapshots will not be used in subsequent builds.
-    SERIAL_NUMBER = 2
+    SERIAL_NUMBER = 3
```

Comment:
This does need to be bumped because the an existing cached build will have unpredictable results since the flags/compilation steps have changed. It's safer to force a clear.